src/NexusMods.App.UI/RightContent/LoadoutGrid/ColumnType.cs

```diff
@@ -10,5 +10,12 @@ public enum ColumnType
     Version,
     Category,
     Installed,
-    Enabled
+    Enabled,
+    
+    // Download
+    DownloadName,
```

Comment:
These aren't Loadout Columns, so they don't belon in `/LoadoutGrid/` We should either subclass some new grid, or make `LoadoutGrid` more generic.

Reply:
Fixed in 39f3c78ec5953ac03080eb47fad59a139fa9b251